app/config/db.js

```diff
@@ -21,8 +21,8 @@ const config = {
 const getConfig = async () => {
   if (process.env.NODE_ENV === 'production') {
     const credential = new DefaultAzureCredential()
-    const token = await credential.getToken('https://ossrdbms-aad.database.windows.net', { requestOptions: { timeout: 1000 } })
-    config.postgresConnectionOptions.password = token.token
+    const { token } = await credential.getToken('https://ossrdbms-aad.database.windows.net', { requestOptions: { timeout: 1000 } })
```

Comment:
would this URL benefit from being in an ENV var instead of hard coded in the config?

Reply:
Good point. I think it's unlikely to change but will move out into ENV.

Reply:
Going to leave this in for now. It's part of one of the templates so will need further investigation if it's like that for a reason.